### PR TITLE
feat: proposed changes from the DEPR WG

### PR DESCRIPTION
**Pull request by deborahgu**
These proposed changes from the deprecation working group are around making sure that every accepted deprecation ticket has an owner responsible for shepherding it through the process.

### CODE REVIEW

File: .gitignore
Comment (sarina) [inline .gitignore:10]: Hi! I recommend that we don't add more editor-specific configurations in .gitignore.

Check out your own personal gitignore! https://tekin.co.uk/2020/03/maintain-a-global-git-ignore-file

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (sarina) [inline oeps/processes/oep-0021-proc-deprecation.rst:295]: So, a bit lower in this document there is a section that says to create an issue on the forums, and it's _not_ optional. I think you should add in this language about seeking an owner to that section.

https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0021-proc-deprecation.html#to-the-open-edx-discourse

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (sarina) [inline oeps/processes/oep-0021-proc-deprecation.rst:305]: In the Product Review Process we have a role called a Coordinator. Perhaps that language would make more sense than "owner"? https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Roles

**Review by sarina — COMMENTED** [review]
Please add a line to the CHANGELOG, too. Thanks!

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (deborahgu) [reply oeps/processes/oep-0021-proc-deprecation.rst:295]: It's a different phase (the non-optional phase is once you have an owner; the optional phase is when you're just noodling around). Let me think about how to clarify that.

File: .gitignore
Comment (deborahgu) [reply .gitignore:10]: Is this a policy? As a matter of best practice I disagree (I think `.gitignore` files that avoid developer fatfinger risk are always good practice), but if this is org-wide policy I certainly will follow it.

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (deborahgu) [reply oeps/processes/oep-0021-proc-deprecation.rst:305]: @dianakhuang, @feanil, does this work for you?

File: .gitignore
Comment (sarina) [reply .gitignore:10]: It's something that I've been asked to do before, I think by @feanil

Comment (feanil) [reply .gitignore:10]: I think it's fine to add `.vscode` as it's a standard IDE home especially given the other ones we have here.  I would be more concerned if it was something that was developer specific such that it would not be useful for anyone or might lead to confusion about why a file isn't getting tracked.

Comment (sarina) [reply .gitignore:10]: Fine with me. I've been stopped from adding it to this repo before.

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (feanil) [reply oeps/processes/oep-0021-proc-deprecation.rst:305]: So it would be  along the lines of: "All active DEPRs must have a coordinator." and "The coordinator is responsible for communicating if they're unable to continue coordinating a deprecation."  I think that's fine with me.

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (deborahgu) [reply oeps/processes/oep-0021-proc-deprecation.rst]: I think the modifications I made to this added some clarity, but I'm still not thrilled with what I wrote.

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (deborahgu) [reply oeps/processes/oep-0021-proc-deprecation.rst]: changes made, along with some rewording.

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (sarina) [reply oeps/processes/oep-0021-proc-deprecation.rst]: I think it's good. One thing you could consider doing is adding a link to the `Announce` section, maybe something like: ```Note: This does not replace the formal `announcement <link_to_announce_ref>_` of the DEPR ticket```. But otherwise I think it's good.

**Review by sarina — APPROVED** [review]
As an author of this OEP I'm marking this as approved from my feedback, but given I'm not currently in the DEPR wg you may wish to seek an approver from the current membership.

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (pdpinch) [inline oeps/processes/oep-0021-proc-deprecation.rst:458]:
```suggestion
ticket's state to *Removing* and make a comment on the issue saying you've done so.
```

**Review by pdpinch — APPROVED** [review]
one small copyedit, otherwise 👍 for me.

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (feanil) [inline oeps/processes/oep-0021-proc-deprecation.rst:311]:
```suggestion
- Ensuring the deprecation work happens
```

Suggestion to reiterate that the coordinator does not have to do the work.

File: oeps/processes/oep-0021-proc-deprecation.rst
Comment (feanil) [inline oeps/processes/oep-0021-proc-deprecation.rst:318]:
```suggestion
When you are ready to coordinate the ticket, post a comment on the
GitHub issue saying you're doing so, and mark yourself as the GitHub
issue's owner.  This can be done even if you don't have write access to 
the ticket by making a comment on the ticket that says ``assign me``.
```

Not required suggestion to help people who don't have access to assign tickets but want to participate.

**Review by feanil — APPROVED** [review]
Couple of small wording suggestions that you can take or leave, either way I think this is good to merge once those conversations are resolved.

**Pull request merged**
